docs/_interface/mining.md

```diff
@@ -1,191 +1,208 @@
 
-When you start up your ethereum node with `geth` it is not mining by default. To start it
-in mining mode, you use the `--mine` command-line flag. The `--miner.threads` parameter can
-be used to set the number parallel mining threads (defaulting to the total number of
-processor cores).
 
-    geth --mine --miner.threads=4
+## CPU Mining with Geth
 
-You can also start and stop CPU mining at runtime using the
-[console](../interface/javascript-console). `miner.start` takes an optional parameter for
-the number of miner threads.
+When Geth is started it is not mining by default. Unless it is specifically instructed to mine, it acts only as a node, not a miner. Geth starts as a (CPU) miner if the `--mine` flag is provided. The `--miner.threads` parameter can
+be used to set the number parallel mining threads (defaulting to the total number of processor cores).
 
-    > miner.start(8)
-    true
-    > miner.stop()
-    true
+```shell
+geth --mine --miner.threads=4
+```
 
-Note that mining for real ether only makes sense if you are in sync with the network
-(since you mine on top of the consensus block). Therefore the eth blockchain
-downloader/synchroniser will delay mining until syncing is complete, and after that mining
-automatically starts unless you cancel your intention with `miner.stop()`.
+CPU mining can also be started and stopped at runtime using the [console](/docs/interface/javascript-console). The command `miner.start` takes an optional parameter for the number of miner threads.
 
-In order to earn ether you must have your **etherbase** (or **coinbase**) address set.
-This etherbase defaults to your [primary account](../interface/managing-your-accounts). If
-you don't have an etherbase address, then `geth --mine` will not start up.
+```js
+miner.start(8)
+ true
+miner.stop()
+ true
+```
 
-You can set your etherbase on the command line:
+Note that mining for real ether only makes sense if you are in sync with the network (since you mine on top of the consensus block). Therefore the Ethereum blockchain downloader/synchroniser will delay mining until syncing is complete, and after that mining automatically starts unless you cancel your intention with `miner.stop()`.
 
-    geth --miner.etherbase '0xC95767AC46EA2A9162F0734651d6cF17e5BfcF10' --mine 2>> geth.log
+Like with GPU mining, an etherbase account must be set. This defaults to the primary account in the keystore but can be set to an alternative address using the `--miner.etherbase` command:
 
-You can reset your etherbase on the console too:
+```shell
+geth --miner.etherbase '0xC95767AC46EA2A9162F0734651d6cF17e5BfcF10' --mine
+```
+If there is no account available the miner will not start. The Javascript console can also be used to reset the etherbase account at runtime:
```

Comment:
If there is no account available, then an account will be created and configured as etherbase automatically.

Reply:
```suggestion
If there is no account available an account wil be created and automatically configured to be the coinbase. The Javascript console can be used to reset the etherbase account at runtime:
```